logging_utilities/thread_context.py

```diff
@@ -0,0 +1,8 @@
+from threading import local
+
+
+class ThreadContext(local):
+    pass
```

Comment:
Add docstring

Reply:
I added a docstring.
In the README I changed to add two chapters describing the new middleware and filter separately.
I also added a new _Use Case_ example to show how to use them together in the context of django.